components/org.wso2.identity.event.common.publisher/src/main/java/org/wso2/identity/event/common/publisher/model/EventContext.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.identity.event.common.publisher.model;
+
+/**
+ * Model Class for Event Context.
+ */
+public class EventContext {
```

Comment:
Not formatted

Reply:
Addressed with https://github.com/wso2-extensions/identity-event-publishers/pull/1/commits/8fa9c3addd4f16e4b35ffb0a8e7c96a28c089382